lib/approvals/writers/custom_writer.rb

```diff
@@ -0,0 +1,13 @@
+module Approvals
```

Comment:
If we don't add the file, are there potentially problems with `lib/approvals/writer.rb`? I'm unsure of how the hook would end up working without a blank template for people who don't want to write their own `CustomeWriter`